app-office/visidata/visidata-2.1.1.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Terminal spreadsheet multitool for discovering and arranging data"
+HOMEPAGE="http://visidata.org"
+SRC_URI="https://github.com/saulpw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-python/python-dateutil[${PYTHON_USEDEP}]"
+DEPEND=""
+BDEPEND="
+	test? (
+		dev-vcs/git
+		dev-python/h5py[${PYTHON_USEDEP}]
+		dev-python/openpyxl[${PYTHON_USEDEP}]
+		dev-python/pandas[${PYTHON_USEDEP}]
+		dev-python/requests[${PYTHON_USEDEP}]
+		$(python_gen_impl_dep sqlite)
+		${RDEPEND}
+	)
+"
+
+python_test() {
+	rm tests/load-http.vd || die "Could not remove network-dependent test."
```

Comment:
I think this won't work if your testing with multiple PYTHON_TARGETS enabled. It will run `python_test` for e.g. python3_7, remove these files, complete the tests, and then run `python_test` for e.g. python3.8 and try to remove those files again, which will fail because those files no longer exist since they were already removed.

Usually, removal of problematic tests is done in the `python_prepare_all` phase instead, which is only run once.

Reply:
yea, this is definitely a fragile approach: I was simply copyng the approach taken in the original ebuild, but I think your suggestion is a good one so I will do that.